services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -179,20 +189,11 @@ class SwitchRulesSpec extends BaseSpecification {
         then: "The corresponding rules are installed on the switch"
         synchronizedRules.installedRules.size() == flowRulesCount
 
-        and: "No missing rules were found after rules validation"
-        Wrappers.wait(WAIT_OFFSET) {
-            with(northboundService.validateSwitchRules(srcSwitch.dpId)) {
-                verifyAll {
-                    properRules.size() == flowRulesCount
-                    missingRules.size() == 0
-                    excessRules.size() == 0
-                }
-            }
+        def actualRules = []
+        Wrappers.wait(RULES_INSTALLATION_TIME) {
+            actualRules = northboundService.getSwitchRules(srcSwitch.dpId).flowEntries.sort { it.cookie }
+            actualRules.size() == defaultPlusFlowRules.size()
         }
-
-        and: "Actual rules are equal to previous ones"
-        def actualRules = northboundService.getSwitchRules(srcSwitch.dpId).flowEntries.sort { it.cookie }
-        actualRules.size() == defaultPlusFlowRules.size()
         [actualRules, defaultPlusFlowRules].transpose().each { actual, expected ->
             verifyAll {
                 actual.cookie == expected.cookie
```

Comment:
Could you please try replacing this comparison with something like
!(actual.properties - expected.properties)

Looks like it could be working.